react/features/chat/functions.ts

```diff
@@ -190,3 +191,17 @@ export function getPrivateNoticeMessage(message: IMessage) {
         recipient: message.messageType === MESSAGE_TYPE_LOCAL ? message.recipient : i18next.t('chat.you')
     });
 }
+
+
+/**
+ * Check if participant is not allowed to send group messages.
+ *
+ * @param {IReduxState} state - The redux state.
+ * @returns {boolean} - Returns true if the participant is not allowed to send group messages.
+ */
+export function isSendGroupChatDisabled(state: IReduxState) {
+    const isJwtGroupChatEnabled = isJwtFeatureEnabled(state, 'send-groupchat', false, false);
+    const { groupChatRequiresPermission } = state['features/dynamic-branding'];
+
+    return groupChatRequiresPermission && !isJwtGroupChatEnabled;
```

Comment:
@damencho  please check this

Reply:
PErhaps this can be written to first check the dynamic brandind flag in order to make it more obvious at first glance?

Reply:
This is exactly what we talked offline :)

Reply:
had same ask from @damencho  :) will do